addons/mail/static/src/components/message/message.xml

```diff
@@ -113,13 +113,13 @@
                                 <t t-if="messageView.messageListViewMessageViewItemOwner and messageView.message.originThread and messageView.message.originThread === messageView.messageListViewMessageViewItemOwner.messageListViewOwner.threadViewOwner.thread and messageView.message.notifications.length > 0">
                                     <div t-att-class="{ 'ms-2': messageView.isInChatWindowAndIsAlignedRight }">
                                         <t t-if="messageView.message.failureNotifications.length > 0">
-                                            <span class="o_Message_notificationIconClickable o-error cursor-pointer text-danger" role="button" tabindex="0" t-on-click="messageView.onClickFailure">
+                                            <span class="o_Message_notificationIconClickable o-error cursor-pointer text-danger" role="button" tabindex="0" t-on-click="messageView.onClickFailure" t-att-title="'Failed Message'">
```

Comment:
`t-att` will not allow translation, why not simply `title`?

also keep `t-on` attributes at the end

Reply:
>t-att
Could I not use ``_t()`` inside the t-att? Otherwise I can make it a compute field in messages or something.

I need t-att because I overwrite the title conditionally in mass_mailing/message.xml

And the reason I do that instead of adding a new condition is that otherwise, I need a link module for mass_mailing&sms and mass_mailing&snail_mail so that I can apply their overwrite in their own respective message.xml (which just changes the icon)

> t-on at the end
Will do

Reply:
Indeed if you need an override the best is to make a field and override the compute in other cases.